server/app/routes/user/index.js

```diff
@@ -0,0 +1,72 @@
+//TODO: GET previous orders
+var router = require('express').Router();
+module.exports = router;
+var _ = require('lodash');
```

Comment:
This might get caught by the Linter if it isn't used. I couldn't find a usecase so far in the code